setup.py

```diff
@@ -10,7 +10,7 @@
 
 setuptools.setup(
     name="userstorage",
-    version="0.5.2",
+    version="0.6.0",
```

Comment:
0.6.0 looks right, but the pr is about 0.5.3?

Reply:
Was to do something similar to https://github.com/oVirt/userstorage/commit/7a78e002eff2ffd5a20dca9dfe285df39c99c856
But as the PR as a whole was a new release, it might have been a bit useless indeed.

Reply:
I used to add the new version as the first commit after the release. 0.x.0 is always a development version, 0.x.1 is the first 0.x release.